relay-server/src/metrics_extraction/sessions.rs

```diff
@@ -4,15 +4,15 @@ use relay_common::{UnixTimestamp, Uuid};
 use relay_general::protocol::{
     AbnormalMechanism, SessionAttributes, SessionErrored, SessionLike, SessionStatus,
 };
-use relay_metrics::{DurationUnit, Metric, MetricNamespace, MetricUnit, MetricValue};
+use relay_metrics::{Metric, MetricNamespace, MetricUnit, MetricValue};
 
 use super::utils::with_tag;
 
 /// Namespace of session metricsfor the MRI.
 const METRIC_NAMESPACE: MetricNamespace = MetricNamespace::Sessions;
 
 /// Current version of metrics extraction.
-const EXTRACT_VERSION: u16 = 2;
+const EXTRACT_VERSION: u16 = 3;
```

Comment:
This version bump will allow us to turn off session metrics extraction in outdated external relays, if there is still a significant amount of `duration` buckets coming from those outdated relays.